src/test/java/com/google/sps/CommentServiceTest.java

```diff
@@ -0,0 +1,87 @@
+package com.google.sps;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import com.google.sps.servlets.CommentService;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+
+public final class CommentServiceTest {
+
+  CommentService commentService = new CommentService();
+
+  /**
+   * A list of Strings should be returned for any public video that has comments. 
+   */
+  @Test
+  public void publicVideoReturnsCommentsList() throws GeneralSecurityException, IOException {
+    String videoId = "v3whnmU619I"; // Public video that has some comments.
+    List<String> actual = commentService.getCommentsFromId(videoId);
+
+    Assert.assertFalse(actual.isEmpty());
```

Comment:
I think the test could be more specific, because right now the commentService.getCommentsFromId() could return a hard coded string instead of calling the api, and it still passes this test (but not other tests of course).

Reply:
same point as Jennifer's (lol)